src/api/useCqlLibraryServiceApi.tsx

```diff
@@ -66,6 +63,30 @@ export class CqlLibraryServiceApi {
       }
     );
   }
+
+  async createVersion(id: string, isMajor: boolean): Promise<void> {
+    return await axios.put(
+      `${this.baseUrl}/cql-libraries/version/${id}?isMajor=${isMajor}`,
+      {},
+      {
+        headers: {
+          Authorization: `Bearer ${this.getAccessToken()}`,
+        },
+      }
+    );
+  }
+
+  async createDraft(cqlLibrary: CqlLibrary): Promise<CqlLibrary> {
```

Comment:
not sure if axios response can directly be cast to model

Reply:
sure we are not using it anyways, let me remove that.